koku/masu/processor/report_summary_updater.py

```diff
@@ -177,12 +177,12 @@ def update_summary_tables(self, start_date, end_date, tracing_id):
         msg = f"Summary processing starting for source {self._provider_uuid}"
         LOG.info(log_json(self._tracing_id, msg))
         start_date, end_date = self._format_dates(start_date, end_date)
-        LOG.info(log_json(tracing_id, f"Using start date: {start_date}"))
-        LOG.info(log_json(tracing_id, f"Using end date: {end_date}"))
+        LOG.debug(log_json(tracing_id, f"Using start date: {start_date}"))
+        LOG.debug(log_json(tracing_id, f"Using end date: {end_date}"))
 
         start_date, end_date = self._updater.update_summary_tables(start_date, end_date)
 
-        msg = f"Summary processing completed for source {self._provider_uuid}"
+        msg = f"Summary processing completed for source {self._provider_uuid} start: {start_date} - end: {end_date}"
```

Comment:
I guess because you added the dates here instead?

Reply:
Yeah, I moved them because we log a whole bunch of stuff in the update_summary_tables, and it was easier to find the range that the summary took place for by search for the `Summary processing completed`. I can turn them back to log.info and just log time in both places.